### ISSUE TITLE
UnboundLocalError: cannot access local variable 'spinner' where it is not associated with a value

### ISSUE THREAD

**Issue by howie6879** [tusharsadhwani/packaged#24]
Hi @tusharsadhwani 

I am packaging a Flask project and encountering an error message

```
UnboundLocalError: cannot access local variable 'spinner' where it is not associated with a value
```

Do you have any solutions for this problem?

**Comment by tusharsadhwani** [comment]
Yup, this is a bug, let me release a fix and I'll get back to you

**Comment by tusharsadhwani** [comment]
I've released `v0.6.1` which should fix that error, but I'm sure there's some other error happening. Let me know if anything else crashes!
